src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1292,6 +1294,7 @@ private void NotifyFeaturesAboutStateChange(DatabaseRecord record, long index)
                         SetUnusedDatabaseIds(record);
                         InitializeFromDatabaseRecord(record);
                         IndexStore.HandleDatabaseRecordChange(record, index);
+                        
```

Comment:
I looks there is no need to touch this file so the changes made here could be reverted.

Reply:
Reverted